scripts/lib/install

```diff
@@ -539,6 +539,10 @@ mkdir -p "$deploy_path"/prod-static/serve
 cp -rT "$deploy_path"/prod-static/serve /home/zulip/prod-static
 chown -R zulip:zulip /home/zulip /var/log/zulip /etc/zulip/settings.py
 
+if ! [ -e /home/zulip/deployments/current/zulip-git-version ]; then
+    su zulip -c '/home/zulip/deployments/current && ./tools/update-git-upstream && ./tools/cache-zulip-git-version'
```

Comment:
Is this missing a `cd` at the start of the quoted command?

Reply:
Ack!  How did this not fail tests, I wonder?